python.md

```diff
@@ -47,7 +47,7 @@ When adopting Graviton2, it is recommended to use recent software versions as mu
 
 Python 2.7 is EOL since January the 1st 2020, it is definitely recommended to upgrade to a Python 3.x version before moving to Graviton2.
 
-Python 3.6 will reach [EOL in December, 2021](https://www.python.org/dev/peps/pep-0494/#lifespan), so when starting to port an application to Graviton2, it is recommended to target at least Python 3.7.
+Python 3.6 reached [EOL in December, 2021](https://www.python.org/dev/peps/pep-0494/#lifespan), so when starting to port an application to Graviton2, it is recommended to target at least Python 3.7.
```

Comment:
According to https://endoflife.date/python, even 3.7 security support ends in a month

Reply:
I'll add info about that as well.